packages/backend/src/plugins/permissions.ts

```diff
@@ -13,5 +16,6 @@ export default async function createPlugin(
     discovery: env.discovery,
     identity: env.identity,
     permissions: env.permissions,
+    adapterFactory: new CasbinAdapterFactory(env.config, env.database),
```

Comment:
Do we need to do the creation of the AdapterFactory here? Can we do the adapter initialization inside `PolicyBuiilder.build`? This way we would not need to export the `CasbinAdapterFactory`.

Reply:
I injected adapterFactory here for testing purpose, but I can think more about that.

Reply:
Done. I moved adapter creation to the PolicyBuiilder.build